### ISSUE TITLE
Sudden slow loading despite reinstall

### ISSUE THREAD

**Issue by DistantStatic** [brave/brave-browser#15708]
## Description 

Brave browser has huge slowdowns suddenly. Issue only occurs in normal browsing mode not during incognito mode. Pages load quickly in other browsers as well as in incognito mode, however during normal browsing mode, browser takes forever to load a page. I have checked in developer tools and it relates to how brave is connection to the internet. Again, I have checked my firewall settings, cleared browsing history for "all time" as well has reinstalled brave and the issue still occurs the same.

## Steps to Reproduce
   1. Load up a regular browsing tab and try to load any site.
   2. Observe

## Actual result:
N/A (can provide if needed)

## Expected result:
Equitable loading times

## Reproduces how often: 
Occurs all the time. Having to use an alternative browser.

## Brave version (brave://version info)
version 1.24.82 Chromium 90.0.4430.93 (64bit)

## Version/Channel Information:
version 1.24.82 Chromium 90.0.4430.93 (64bit)
## Other Additional Information:
Brave shields don't even get to activate. Issue does not occur with other browsers. Reinstall does not help.

## Miscellaneous Information:
Issue is not site related. Same sites/queries attempted on same PC with multiple different browsers and were successful with expected loading times. Brave was the only one having issues.

**Comment by ryanbr** [comment]
Can you try removing the existing profile, and retest. How are testing performance to compare load times? @DistantStatic

**Comment by DistantStatic** [comment]
I tested performance by loading static, non internet documents like PDF. They opened just fine. I also compared speeds with 2 different browsers, Edge and Firefox, and both were able to load multiple internet destinations in the time it took Brave to load 1. Woke up this morning and issue is no longer occurring so unfortunately I can't test the new profile. The issue persisted past restarts, reinstalls, browsing cache clear, etc... Not sure what made it get back to normal, but I will close this for now as I can no longer replicate the issue.
